### DISCUSSION TITLE
使用 umamusume-localify 导致无法识别

### DISCUSSION THREAD

**Discussion by mtt-konan** [NateScarlet/auto-derby#312] [Q&A] [answered]
是分辨率不够吗，画面设置成标准化也是这样，版本auto-derby-1.25.1

![p1](https://user-images.githubusercontent.com/102540842/172138942-0266422c-f3b1-42d6-8d11-502bb9f19847.jpg)

![p2](https://user-images.githubusercontent.com/102540842/172138946-301d72c0-67a1-4bae-a637-d7d9a35e3396.jpg)

**Comment by NateScarlet** [comment, marked as answer]
游戏改版了？ 怎么比赛上面只有场馆

**Reply by NateScarlet** [reply]
建了 #313
